holt-winters.c

```diff
@@ -38,9 +38,9 @@ void HoltWinters (
 
           /* return values */
           double *SSE,          // The final sum of squared errors achieved in optimizing
-          double *level,        // Estimated values for the level component (size xl - t + 1)
-          double *trend,        // Estimated values for the trend component (size xl - t + 1)
-          double *season        // Estimated values for the seasonal component (size xl - t + 1)
+          double *level,        // Estimated values for the level component (size xl - t + 2)
```

Comment:
I think the root cause is this:

https://github.com/pierre/holt-winters/blob/3fddab1aed05762d257252d5a9dbf9b0e5c4c8d6/holt-winters.c#L57